homeassistant/components/nobo_hub/climate.py

```diff
@@ -29,12 +28,10 @@
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util import dt
 
 from .const import (
-    ATTR_OVERRIDE_ALLOWED,
     ATTR_SERIAL,
-    ATTR_TARGET_ID,
-    ATTR_TARGET_TYPE,
```

Comment:
Are these constants used elsewhere? If not, they should be deleted in `const.py`.

Reply:
No, they are not. Thanks - deleted.